okhttp/src/main/java/com/squareup/okhttp/MediaTypes.java

```diff
@@ -0,0 +1,108 @@
+package com.squareup.okhttp;
+
+/**
+ * Constants for commonly used media types.
+ *
+ * @see com.squareup.okhttp.MediaType
+ */
+public abstract class MediaTypes {
```

Comment:
Special class for constants is not a good OOP design.. But okay, if you do so, please make sure that the class is not instantiable (private constructor), here is good thread about storing constants in Java http://stackoverflow.com/questions/320588/interfaces-with-static-fields-in-java-for-sharing-constants